portfolio/src/main/webapp/assets/js/script.js

```diff
@@ -16,99 +16,102 @@
  * Creates images for photography gallery
  */
 
-document.addEventListener("DOMContentLoaded", function () {
-  addFilmImages();
-  getComments();
+document.addEventListener("DOMContentLoaded", function() {
+    addFilmImages();
+    getComments();
 });
 
+function zoomOut() {
+    console.log("trigger");
```

Comment:
Do you need this?

Reply:
sorry! these always slip through the cracks soomehow